src/test/java/org/gaul/s3proxy/CrossOriginResourceSharingRuleTest.java

```diff
@@ -39,9 +39,10 @@ public void setUp() throws Exception {
                         "https://example\\.cloud"),
                 Lists.newArrayList("GET", "PUT"),
                 Lists.newArrayList("Accept", "Content-Type"),
+                Lists.newArrayList(),
                 "true");
         // CORS disabled
-        corsOff = new CrossOriginResourceSharing(null, null, null, null);
+        corsOff = new CrossOriginResourceSharing(null, null, null, null, null);
```

Comment:
Is there any kind of test that can exercise how this should work when it is configured/not configured?  s3-tests might have some functionality so we might just need to enable tests there.  Currently CI disables all the cors tests:

```
src/test/resources/run-s3-tests.sh:./virtualenv/bin/nosetests -a '!fails_on_s3proxy,!appendobject,!bucket-policy,!cors,!encryption,!fails_strict_rfc2616,!lifecycle,!object-lock,!policy,!policy_status,!s3select,!s3website,!tagging,!user-policy,!versioning'
```

We can decouple testing from this PR if you think it makes sense.

Reply:
I don't see much value in including a test to check the exposed headers from this file since it would just be a simple passthrough-existance check . I can, however, investigate adding some better tests for `Access-Control-Expose-Headers` in a follow-on PR if thats alright.

I will also update the docs related to the new `s3proxy.cors-exposed-headers` (`S3PROXY_CORS_EXPOSED_HEADERS`) properties in that PR if that works for you.

Reply:
Yeah let's add the properties at least.  I opened #675 to track possible testing but let's not block this PR on that.

Reply:
Oh I see you already added the Docker environment variable.